src/components/ChatApp/Settings/ShareOnSocialMedia.js

```diff
@@ -3,6 +3,7 @@ import Translate from '../../Translate/Translate.react';
 import FontIcon from 'material-ui/FontIcon';
 import RaisedButton from 'material-ui/RaisedButton';
 import urls from '../../../utils/urls';
+import PropTypes from 'prop-types';
 
 export default class ShareOnSocialMedia extends Component {
   constructor(props) {
```

Comment:
Use functional component here

Reply:
Here, component has state, so lets keep it this way only